styles/perplexity/catppuccin.user.css

```diff
@@ -66,197 +67,358 @@
+    }
+
+    [class*="hover:bg-super"]:hover {
+      background-color: @accent-color !important;
+    }
+
+    [class*="hover:text-super"]:hover,
+    [class*="hover:text-superAlt"]:hover {
+      color: @accent-color !important;
     }
-    .dark .dark\:outline-superDark {
-      outline-color: @accent-color;
+
+    [class*="hover:text-textMain"]:hover {
+      color: @text !important;
+    }
+
+    [class*="hover:text-textOff"]:hover {
+      color: @subtext0 !important;
     }
-    .text-super {
-      color: @accent-color;
+
+    [class*="hover:text-white"]:hover {
+      color: @text !important;
     }
 
-    // prose
-    .prose,
-    :is(:where(.dark) .dark\:prose-invert) {
-      --tw-prose-body: @text;
-      --tw-prose-headings: @text;
-      --tw-prose-lead: @text;
-      --tw-prose-links: @blue;
-      --tw-prose-bold: @text;
-      --tw-prose-counters: @text;
-      --tw-prose-bullets: @text;
-      --tw-prose-hr: @text;
-      --tw-prose-quotes: @text;
-      --tw-prose-quote-borders: @text;
-      --tw-prose-captions: @text;
-      --tw-prose-kbd: @text;
-      --tw-prose-kbd-shadows: @text;
-      --tw-prose-code: @text;
-      --tw-prose-pre-code: @text;
-      --tw-prose-pre-bg: @base;
-      --tw-prose-th-borders: @text;
-      --tw-prose-td-borders: @text;
+    .dark\:divide-borderMainDark\/50:where(
+        [data-color-scheme="dark"] .dark\:divide-borderMainDark\/50,
+        [data-color-scheme="dark"] .dark\:divide-borderMainDark\/50 *
+      )
+      > :not([hidden])
+      ~ :not([hidden]),
+    .divide-borderMain\/50 > :not([hidden]) ~ :not([hidden]) {
+      border-color: @surface1 !important;
+    }
+
+    .focus\:\!border-backgroundDark:focus {
+      border-color: @mantle !important;
+    }
+
+    .focus\:bg-background:focus {
+      background-color: @base !important;
+    }
+
+    .focus\:text-super:focus {
+      color: @accent-color !important;
+    }
+
+    *[class*="ring"] {
+      --tw-ring-color: @surface2 !important;
+      --tw-ring-offset-color: @mantle !important;
+    }
+
+    /* Additional fixes: */
+
+    /* citations */
+    .citation:hover * {
+      color: @crust !important;
+    }
+    /* switch knob in settings */
+    .data-\[state\=checked\]\:before\:\!bg-super[data-state="checked"]::before {
+      background-color: @accent-color !important;
+    }
+    /* switch in main input */
+    #copilot-toggle > div::before {
+      background-color: @overlay1;
+    }
+
+    /* credits on image view */
+    #__next
+      > div:nth-child(26)
+      > div:nth-child(2)
+      > div
+      > div
+      > div.gap-md.mx-md.py-md.border-borderMainDark.flex.h-\[74px\].items-center.justify-between.border-b-2
+      > div.gap-x-md.flex.items-center
+      > a
+      > div
+      > div
+      > div.line-clamp-1.break-all.transition-all.duration-300.white.font-sans.text-sm.text-white.selection\:bg-super\/50.selection\:text-textMain.dark\:selection\:bg-superDuper\/10.dark\:selection\:text-superDark {
```

Comment:
Surely this could have just been themed by selecting only the `.text-white` class?

Reply:
This is because this usage of text-white is on a different background than the other ones, I did theme text-white